src/routes/EntryPointRoute.tsx

```diff
@@ -56,18 +68,24 @@ export default function EntryPointRoute(
       };
     }, [data.queries]);
 
+    if (!loadedEntryPoint) throw loadEntryPoint();
+    const resource = loadedEntryPoint.root;
     const Component = resource.getModuleIfRequired();
     if (Component) {
       return <Component {...data} />;
     }
     throw resource.load();
   };
-  Hoc.displayName = `EntryPointRoute(${resource.getModuleId()})`;
+  Hoc.displayName = `EntryPointRoute`;
```

Comment:
If we use a JSResource for the lazy entry point case I think we'll be able to use its moduleId as an identifier here too